roboteq_diff_driver/src/driver.cpp

```diff
@@ -884,82 +912,83 @@ ROS_DEBUG("");
 int MainNode::run()
 {
 
-	ROS_INFO("Beginning setup...");
-
-	serial::Timeout timeout = serial::Timeout::simpleTimeout(1000);
-	controller.setPort(port);
-	controller.setBaudrate(baud);
-	controller.setTimeout(timeout);
-
-	// TODO: support automatic re-opening of port after disconnection
-	while ( ros::ok() )
-	{
-		ROS_INFO_STREAM("Opening serial port on " << port << " at " << baud << "..." );
-		try
-		{
-			controller.open();
-			if ( controller.isOpen() )
-			{
-				ROS_INFO("Successfully opened serial port");
-				break;
-			}
-		}
-		catch (serial::IOException e)
-		{
-			ROS_WARN_STREAM("serial::IOException: " << e.what());
-		}
-		ROS_WARN("Failed to open serial port");
-		sleep( 5 );
-	}
-
-	cmdvel_setup();
-	odom_setup();
+  ROS_INFO("Beginning setup...");
+
+  serial::Timeout timeout = serial::Timeout::simpleTimeout(1000);
+  controller.setPort(port);
+  controller.setBaudrate(baud);
+  controller.setTimeout(timeout);
+
+  while ( ros::ok() )
+  {
+    if (open_serial())
+    {
+      break;
+    }
+    else
+    {
+      sleep( 5 );
```

Comment:
Choose a spacing style?

Reply:
lol tell that to the original author.. i was tempted to run a formatter on this file but held back because if we ever want to PR this back upstream that'd be a super noisy diff.